### DISCUSSION TITLE
gatsby-source-wordpress warning about  'createRoot' is not exported

### DISCUSSION THREAD

**Discussion by xy-lin** [gatsbyjs/gatsby#33841] [Help]
Hi guys,
I have got following warning when do `gatsby build` or `gatsby develop`:
warn ./node_modules/gatsby-source-wordpress/dist/gatsby-browser.js
Attempted import error: 'createRoot' is not exported from 'react-dom' (imported as 'ReactDOM').

package info as follow:

Binaries:
    Node: 16.13.0 - /usr/local/bin/node
    npm: 8.1.0 - /usr/local/bin/npm
npmPackages:
    gatsby: ^4.1.0
    gatsby-plugin-gatsby-cloud: ^4.1.0
    gatsby-plugin-image: ^2.1.0 
    gatsby-plugin-manifest: ^4.1.0
    gatsby-plugin-react-helmet: ^5.1.0 
    gatsby-plugin-sharp: ^4.1.0 
    gatsby-source-filesystem: ^4.1.0
    **gatsby-source-wordpress: ^6.1.0**
    gatsby-transformer-sharp: ^4.1.0

react: ^17.0.2
react-dom: ^17.0.2

Is it a problem of React or gatsby-source-wordpress? Any idea? Many thanks!

**Comment by LekoArts** [comment]
Please see: https://github.com/gatsbyjs/gatsby/discussions/33856#discussioncomment-1586716

**Reply by xy-lin** [reply]
The warning message  'createRoot' is not exported from 'react-dom' is still there. After following the clean install with --legacy-peer-deps flag. The only way to make it disappear for now is to hack the gatsby-browser.js as mentioned here: https://github.com/gatsbyjs/gatsby/discussions/33856#discussioncomment-1591121

**Reply by sahandnayebaziz** [reply]
This is currently happening to me too and totally slowing down our netlify builds

**Reply by zamson** [reply]
+1
